staging/src/k8s.io/client-go/applyconfigurations/admissionregistration/v1/mutatingwebhookconfiguration.go

```diff
@@ -232,12 +232,12 @@ func (b *MutatingWebhookConfigurationApplyConfiguration) WithFinalizers(values .
 	return b
```

Comment:
I'd suggest blocking this field here to drop it from apply configurations now:

```
func blocklisted(t *types.Type, member types.Member) bool {
	if objectMeta.Name == t.Name && member.Name == "ManagedFields" {
		return true
	}
	if objectMeta.Name == t.Name && member.Name == "SelfLink" {
		return true
	}
```

Reply:
I didn't realize it'd be that easy.